{"name": "获取同步根组id", "status": "failed", "statusDetails": {"message": "AssertionError", "trace": "self = <bms.testcases.test_02_usersync.TestUserSync object at 0x000001601A27CF40>\ncaseid = 'usersync_get_syncgroupid_01'\ncasedata = {'base_url': '${config(host)}', 'caseid': 'usersync_get_syncgroupid_01', 'description': '获取同步根组id', 'epic': '用户管理', ...}\n\n    @pytest.mark.parametrize('caseid, casedata', [(caseid, casedata) for caseid, casedata in\n                                                  readcase.ReadCase().get_case_dict(\n                                                      ROOT_DIR + 'bms/data/usersync/usersync_get_syncgroupid.yaml').items()])\n    def test_usersync_get_syncgroupid(self, caseid, casedata):\n        handleallure.allure_display(casedata)\n        logging.info(\"==========开始执行用例：%s==========\", caseid)\n>       runcase.excute_case(casedata)\n\nbms\\testcases\\test_02_usersync.py:165: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\ncommon\\runcase.py:55: in excute_case\n    checkresult.check_result(hope_result, recv_data, recv_code)\ncommon\\checkresult.py:31: in check_result\n    assert_response(value, real_res)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nhope_res = {'msg': 'success', 'statusCode': 0}\nreal_res = {'msg': '找不到对应的组织机构', 'statusCode': 1114886}\n\n    def assert_response(hope_res, real_res):\n        \"\"\"\n        返回结果校验 -全返回校验\n        :param hope_res:\n        :param real_res:\n        :return:\n        \"\"\"\n        try:\n            with allure.step(\"返回值校验\"):\n                allure.attach(name=\"期望返回值\", body=str(hope_res))\n                allure.attach(name='实际返回值', body=str(real_res))\n                if isinstance(hope_res, (dict, list)) and isinstance(real_res, (dict, list)):\n                    flag = handledict.cmp_dict(hope_res, real_res)\n>                   assert flag\nE                   AssertionError\n\ncommon\\checkresult.py:83: AssertionError"}, "description": "获取同步根组id", "steps": [{"name": "执行关联接口", "status": "passed", "steps": [{"name": "执行当前接口：同步任务列表搜索-无搜索条件", "status": "passed", "attachments": [{"name": "请求方法：", "source": "41374da1-9084-4f06-a9c8-af38a63402fb-attachment.attach"}, {"name": "请求地址", "source": "4e8333d4-5826-4c9d-9f3b-c643a9261a37-attachment.attach"}, {"name": "请求头", "source": "695863d7-8df7-475a-9598-84a3871d2f2b-attachment.attach"}, {"name": "请求参数", "source": "aa5881f5-5341-4e43-93ab-880c69f02637-attachment.attach"}, {"name": "请求结果", "source": "0fe0a506-2b42-4fef-842a-f5d33fb3111e-attachment.attach"}], "start": 1687868048501, "stop": 1687868048578}, {"name": "结果校验", "status": "passed", "steps": [{"name": "状态码校验", "status": "passed", "attachments": [{"name": "期望状态码", "source": "0e802c50-b0ca-4c49-a343-46ef9e7847f3-attachment.attach"}, {"name": "实际状态码", "source": "5cb4b07b-02dd-43cf-8e43-ec4ab4e20f95-attachment.attach"}], "start": 1687868048578, "stop": 1687868048579}, {"name": "返回值校验", "status": "passed", "attachments": [{"name": "期望返回值", "source": "6c27694c-cd6c-4bc2-a9fa-29ef8e7df305-attachment.attach"}, {"name": "实际返回值", "source": "d3df94eb-e95e-4471-b2ed-dd6389088530-attachment.attach"}], "start": 1687868048579, "stop": 1687868048580}], "start": 1687868048578, "stop": 1687868048580}], "start": 1687868048476, "stop": 1687868048580}, {"name": "执行当前接口：获取同步根组id", "status": "passed", "attachments": [{"name": "请求方法：", "source": "588dfcd7-8da3-4157-b77f-7a2397d7b7f2-attachment.attach"}, {"name": "请求地址", "source": "8172edc5-f659-48c8-92cd-20679e66b4af-attachment.attach"}, {"name": "请求头", "source": "791d5a73-cd4a-4ba5-9a83-28e48da68ac3-attachment.attach"}, {"name": "请求参数", "source": "6c80b62b-db47-4412-ad7f-0d51bc5234e0-attachment.attach"}, {"name": "请求结果", "source": "ead1c5b9-071b-4ef1-81e2-cf87b1e84680-attachment.attach"}], "start": 1687868048582, "stop": 1687868048649}, {"name": "结果校验", "status": "failed", "statusDetails": {"message": "AssertionError\n", "trace": "  File \"E:\\Wondersoft_ApiTest\\common\\checkresult.py\", line 31, in check_result\n    assert_response(value, real_res)\n  File \"E:\\Wondersoft_ApiTest\\common\\checkresult.py\", line 83, in assert_response\n    assert flag\n"}, "steps": [{"name": "状态码校验", "status": "passed", "attachments": [{"name": "期望状态码", "source": "4f0c165f-e4e1-48a8-85c5-f93c0054b1a7-attachment.attach"}, {"name": "实际状态码", "source": "942369b4-e0d4-46c9-b26b-ba32875fde5b-attachment.attach"}], "start": 1687868048649, "stop": 1687868048650}, {"name": "返回值校验", "status": "failed", "statusDetails": {"message": "AssertionError\n", "trace": "  File \"E:\\Wondersoft_ApiTest\\common\\checkresult.py\", line 83, in assert_response\n    assert flag\n"}, "attachments": [{"name": "期望返回值", "source": "75479712-a184-46ea-b7fa-dcc440c89200-attachment.attach"}, {"name": "实际返回值", "source": "f81ff434-fb73-4e4f-bcb8-fcc7b52ec18a-attachment.attach"}], "start": 1687868048650, "stop": 1687868048651}], "start": 1687868048649, "stop": 1687868048651}], "attachments": [{"name": "log", "source": "751b1deb-238b-48b7-aac8-13fca139b433-attachment.txt", "type": "text/plain"}], "start": 1687868048469, "stop": 1687868048651, "uuid": "52e3f47e-8977-414f-8586-63edfbad73d1", "historyId": "c076d0bb657b7fadcb6dc7f1415a3243", "testCaseId": "8a41af8eca098c4295d5e872d129839c", "fullName": "bms.testcases.test_02_usersync.TestUserSync#test_usersync_get_syncgroupid", "labels": [{"name": "epic", "value": "用户管理"}, {"name": "feature", "value": "用户同步管理-用户同步"}, {"name": "story", "value": "编辑，获取同步根组id接口"}, {"name": "parentSuite", "value": "bms.testcases"}, {"name": "suite", "value": "test_02_usersync"}, {"name": "subSuite", "value": "TestUserSync"}, {"name": "host", "value": "OFYI7JX9DURTDV6"}, {"name": "thread", "value": "14564-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "bms.testcases.test_02_usersync"}]}